docs/jwst/tweakreg/README.rst

```diff
@@ -16,7 +16,7 @@ Source Detection
 If the ``meta.tweakreg_catalog`` attribute of input data models is a non-empty
 string and ``use_custom_catalogs`` is `True`, then it will be interpreted
 as a file name of a user-provided source catalog. The catalog must be in a
-format automatically recognized by :py:meth:`~astropy.table.Table.read`.
+format automatically recognized by :ref:`astropy.table.Table.read <astropy:read_write_tables>`.
 
 When the ``meta.tweakreg_catalog`` attribute of input data models is `None` or
 an empty string, then the ``tweakreg`` step will attempt to detect sources in the
```

Comment:
I can't comment on an unmodified line, but I noticed that none of the photutils classes are clickable in this paragraph, and maybe they should be?  I think you'd just need to add a `~`

Reply:
Added below.